paasta_tools/cli/cmds/logs.py

```diff
@@ -69,11 +70,32 @@
 from paasta_tools.utils import get_log_name_for_service
 
 
-DEFAULT_COMPONENTS = ["build", "deploy", "monitoring", "oom", "stdout", "stderr"]
+DEFAULT_COMPONENTS = ["stdout", "stderr"]
+TEXTWRAP_LIST_ITEM_PREFIX = "*"
 
 log = logging.getLogger(__name__)
 
 
+class PerLineWrapFormatter(argparse.RawDescriptionHelpFormatter):
+    """Custom argparse formatter that wraps each line in argument help text
+    individually, instead of wrapping the entire help text at once. This allows
+    us to maintain newlines where we want them.
+    """
+
+    def _split_lines(self, text, width):
+        wrapped_lines = []
+        for line in text.splitlines():
+            # add some indents if we see a list in help text
+            if line.startswith(TEXTWRAP_LIST_ITEM_PREFIX):
```

Comment:
This feels like we're blurring the boundaries of the formatter class. If you had to describe this feature (the extra list formatting) in the class docstring, does it make sense? At the very least, TEXTWRAP_LIST_ITEM_PREFIX should be a member of this class, and its length should be used the next line, at `line[1:]`, rather than assuming its length is 1.

I agree the component list is less readable in the help text without the list formatting/indenting. To me, though, it doesn't seem worth this complexity, and keeping the list of components in the epilog seems good enough.

Reply:
I prefer how it looks, but I rolled this change back anyway.